pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -90,6 +90,7 @@ type DomainManager interface {
 	SignalShutdownVMI(*v1.VirtualMachineInstance) error
 	MarkGracefulShutdownVMI(*v1.VirtualMachineInstance) error
 	ListAllDomains() ([]*api.Domain, error)
+	ListAllDomainsWithRuntimeInfo() ([]*api.Domain, error)
```

Comment:
We  can't just introduce this new endpoint without thinking about virt-handler backward compatibility. I know it is more work on the unit-test side to change `ListAllDomains` to return everything, but I think we need to go this route, or at least  make virt-handler fall back to `ListAllDomains` (not my preference).